include/hipSYCL/glue/generic/hiplike/hiplike_kernel_launcher.hpp

```diff
@@ -587,8 +662,9 @@ class hiplike_kernel_launcher : public rt::backend_kernel_launcher
 
           if constexpr (type == rt::kernel_type::basic_parallel_for) {
 
-            __hipsycl_launch_kernel(
-                hiplike_dispatch::parallel_for_kernel<KernelName>,
+            __hipsycl_invoke_kernel(
+                hiplike_dispatch::parallel_for_kernel<KernelName>, KernelName,
```

Comment:
Soo.. I checked all `__hipsycl_invoke_kernel()` calls again, and it seems that this is what `clang-format` thinks is optimal. Apparently it just tries to put all arguments into one line until it would exceed the maximum line width, then just inserts a line break and continues with the next line.
I can change this manually, but it will probably be changed back the next time somebody auto-formats that code.